controlpanel/api/models/s3bucket.py

```diff
@@ -98,9 +98,7 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
         if is_create:
-            bucket = cluster.create_bucket(self.name, self.is_data_warehouse)
-            if bucket:
-                self.location_url = bucket['Location']
```

Comment:
So we *definitely* don't use this `location_url` field, I seem to remember you mentioned checking the DB and it's always blank or something 🤷‍♂

Reply:
It is always blank, because we never saved the model after we set the field (ever since jmoz wrote it originally). But it isn't used anywhere anyway.

Reply:
Yes, it's never saved to the database, ever since jmoz first wrote it. And it's never used anyway.